src/common/components/Select.tsx

```diff
@@ -3,22 +3,26 @@ import React, { ComponentProps } from 'react'
 type OptionValue = React.OptionHTMLAttributes<HTMLOptionElement>['value']
 
 interface Props<T extends OptionValue> extends Omit<ComponentProps<'select'>, 'onChange'> {
+  selectRef?: React.Ref<HTMLSelectElement>
```

Comment:
Thoughts on bumping minimum React version to React 19? Pretty new for a massive hooks library, but then we'd be able to use a lot of the fun improvements like [exposing `ref` as a prop](https://react.dev/blog/2024/04/25/react-19#ref-as-a-prop).

Reply:
I'm into it but I think it should probably be a separate PR